src/woo_publications/publications/admin.py

```diff
@@ -698,6 +698,13 @@ def delete_queryset(
                 )
             )
 
+    def formfield_for_foreignkey(self, db_field, request, **kwargs):
+        if db_field.name == "publicatie":
+            kwargs["queryset"] = Publication.objects.filter(
+                ~models.Q(publicatiestatus=PublicationStatusOptions.revoked)
+            )
```

Comment:
I think this drops potential application of `limit_choices_to` at the model field level, and may drop the ordering from the admin. Probably safer to repeat the default implementation and modify that:

```suggestion
            default_qs = self.get_field_queryset(db, db_field, request)
            assert default_qs is not None
            kwargs["queryset"] = default_qs.exclude(publicatiestatus=PublicationStatusOptions.revoked)
```

Also note the usage of `.exclude` instead of manually building that using `~models.Q(...)` which is much less readable.